azure-pipelines.yml

```diff
@@ -20,7 +20,7 @@ resources:
     endpoint: nhs-digital-gp-it-futures
 
 pool:
-  vmImage: 'ubuntu-latest'
+  vmImage: 'Ubuntu-20.04'
```

Comment:
Are we not already getting this version using `-latest`?

Reply:
Nope, latest is still Ubuntu-16.04' for now. In theory we are automatically moving to it soon, but it's already 1 month late. So I have forced this to reduce the warnings from the pipeline

Reply:
Fair enough